src/main/java/cceclipseplugin/editor/listeners/SaveExecutionListener.java

```diff
@@ -1,35 +0,0 @@
-package cceclipseplugin.editor.listeners;
-
-import org.eclipse.core.commands.ExecutionEvent;
-import org.eclipse.core.commands.ExecutionException;
-import org.eclipse.core.commands.IExecutionListener;
-import org.eclipse.core.commands.NotHandledException;
-
-import cceclipseplugin.core.PluginManager;
-
-public class SaveExecutionListener implements IExecutionListener {
```

Comment:
This wasn't being used at all and was a thing from way back when I was trying to make it so that saving pulls, diffs, and sends changes.